contracts/mockup/FeeSharingProxyMockup.sol

```diff
@@ -11,7 +11,7 @@ contract FeeSharingProxyMockup is FeeSharingProxy {
 
 	TestData public testData;
```

Comment:
just wonder if we can get rid of this mockup and replace it by for waffle mockup?